tests/model/stepfunctions/test_state_machine_generator.py

```diff
@@ -148,6 +148,37 @@ def test_state_machine_with_unsupported_event_source(self):
         with self.assertRaises(InvalidEventException):
             StateMachineGenerator(**self.kwargs).to_cloudformation()
 
+    def test_state_machine_with_alias_as_event_source_target(self):
```

Comment:
Great tests! Thanks for adding. Can we also add transform tests defined [in this guide](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-statemachine-statemachineeventsource.html). Would give an idea of what the SAM template and the transformed template looks like.